torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -199,8 +199,11 @@ void assignRNGOffset(Fusion* fusion) {
 }
 
 // Dump expr string if enable lower_verbose
-void dumpExprsIfEnabled(const std::vector<Expr*>& exprs, std::string msg_pre) {
-  if (isDebugDumpEnabled(DebugDumpOption::LowerVerbose)) {
+void dumpExprsIfEnabled(
+    const std::vector<Expr*>& exprs,
+    std::string msg_pre,
+    bool force_enable = false) {
```

Comment:
I feel this boolean flag is very helpful for debugging. When I want to see something, I just manually add a `, true` to the place I want to print.